client/src/pages/CategoryPage.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { Header } from "../components/Header";
+import { ImagePreview } from "../components/ImagePreview";
+import { Navbar } from "../components/Navbar";
+
+const Container = styled.div`
+  padding: 60px 10px;
+`;
```

Comment:
Maybe try adding a general `Container` component, so you don't have to create a new Container Component for every Component